### PR TITLE
feat: statistic card

### PR DESCRIPTION
**Pull request by kelvin3127**
Statistic Card

### Description
A Statistic Card component
Includes theme styling
More can be added in future commits
Includes Storybook

--- Icons will be coming from icons component, temporarily just text.

### Testing Instructions

Open terminal and run "npm run dev"
Open application in browser with local host.
Import statistics card component and add to the template.
Statistics card component Props include:
theme - Changes the theme for the dropdown.
cardType - Changes the type of Statistic Card you would like, currently have only 1
data - The data being passed in
If no theme are is selected component will have it's own default theme.

### Testing Instructions Storybook

Open terminal and run "npm run storybook"
Open storybook in browser "http://localhost:6006/".
Select theme you would like to view

### Screenshots
Three Components

Storybook
Three Themes

### CODE REVIEW

File: frontend/src/components/StatisticCard/StatisticCard.vue
Comment (YoussefLaunchUp) [inline frontend/src/components/StatisticCard/StatisticCard.vue:1]: rename this to index.vue

File: frontend/src/components/StatisticCard/StatisticCard.vue [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        return { themeType, themeChildType, type, statData, lastUpdate, calculateLastTimeUpdate}
+    },
+})
+</script>
+
+<template>
+    <div>
+        <div class="flex flex-col w-1/2 rounded-md border drop-shadow-md border-black" :class="themeType">
+            <div class="flex flex-row">
+                <p class="flex-grow mx-4 my-2">{{ cardType }} Icon</p>
```
Comment (YoussefLaunchUp) [inline frontend/src/components/StatisticCard/StatisticCard.vue]: Although you are not using the icons right now, this component should be able to accept any icon independent from the type.

**Review by YoussefLaunchUp — CHANGES_REQUESTED** [review]

File: frontend/src/components/StatisticCard/StatisticCard.vue (hunk 1 above)
Comment (kelvin3127) [reply frontend/src/components/StatisticCard/StatisticCard.vue]: Changes made, renamed Component to index.vue and added new prop for icon type to allow user choice of icon

**Review by neli-rashkova — APPROVED** [review]
Good work!

**Review by YoussefLaunchUp — APPROVED** [review]
Great work

**Pull request merged**